stdlib/public/Concurrency/TaskLocal.swift

```diff
@@ -102,6 +102,7 @@ public final class TaskLocal<Value: Sendable>: Sendable, CustomStringConvertible
     self.defaultValue = defaultValue
   }
 
+  @usableFromInline
```

Comment:
`@usableFromInline` adds a public symbol that won't be available on older platforms. Can you make this `@alwaysEmitIntoClient`?

Reply:
Ok will do, I hope backdeploy will understand that hmm.

Reply:
Okey cool it does :)